README.md

```diff
@@ -11,3 +11,38 @@ TODO: Add detailed app description and demo
 ![architecture diagram](images/architecture-diagram.png)
 [Image Link](https://excalidraw.com/#json=1c_DrTFZCSGprCvJNBYHn,B0sXrISCY8YdKgiTqy9xDA)
 _revised 9/4/2024_
+
+The structure of the event database is as follows:
+ ```json
+ {
+  "Event": {
+    "Name": "New Years Eve Party",
+    "Photo": "https://stackoverflow.com/questions/46585330/firestore-security-rules-for-public-and-private-fields",
+    "Description": "Get Ready for NYE 2025, dress classy and dance all night",
+    "Category": "Party",
```

Comment:
Can you specify all the categories we will have available? I believe we said we will only have 3-4 for v1.

Reply:
Is there a way to specify it when the field only accepts one category?

Cannot add comments in json